docs/CONFIG.md

```diff
@@ -4687,6 +4688,12 @@ SendOnly = false # Default
 ```
 SendOnly limits usage to sending transaction broadcasts only. With this enabled, only HTTPURL is required, and WSURL is not used.
 
+### Priority
+```toml
+Priority = 0 # Default
+```
+Priority of the node in the pool, only takes effect if `SelectionMode` is `PriorityLevel`
```

Comment:
Do you think it would also make sense to support using this field as a tie-breaker when using some other selection modes? e.g. `HighestHead` and `TotalDifficulty`

Reply:
Sure, I believe it will give more flexibility in configuring nodes. Added in [`ed45428` (#9329)](https://github.com/smartcontractkit/chainlink/pull/9329/commits/ed45428cd4949ff01e01fa059f148e31bbfaa142)